src/components/AttachmentPicker/index.native.tsx

```diff
@@ -237,7 +237,7 @@ function AttachmentPicker({type = CONST.ATTACHMENT_PICKER_TYPE.FILE, children, s
 
     const validateAndCompleteAttachmentSelection = useCallback(
         (fileData: FileResponse) => {
-            if (fileData.width === -1 || fileData.height === -1) {
+            if (fileData.width === -1 || fileData.height === -1 || (fileData.height === 0 && fileData.width === 0)) {
```

Comment:
```suggestion
            if (fileData.width <= 0 && fileData.height <= 0) {
```

Can you also add a comment with this? Useful info to share
> Yeah, on ios native the image width/height for corrupted image is 0 and on android native it is -1.